airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -475,6 +475,13 @@ class Tickets(SourceZendeskIncrementalExportStream):
 
     response_list_name: str = "tickets"
 
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
```

Comment:
any reason not to use the TypeTransformer [here](https://docs.airbyte.com/connector-development/cdk-python/schemas#type-transformation)?

Reply:
Changed to use TypeTransformer!